Cargo.toml

```diff
@@ -35,3 +35,9 @@ examples = []
 [[example]]
 name = "hello-world"
 required-features = ["examples"]
+
+# Enable LTO to workaround https://github.com/rust-lang/rust/issues/62785
+[profile.dev]
+lto = true
+[profile.release]
+lto = true
```

Comment:
This is no longer needed, right? (https://github.com/rust-lang/compiler-builtins/pull/317/files)

Btw., I don't think what they do is correct. There is no guarantee that floating point registers are actually initialized, so they should not define the symbol. At the same time, this is only a safety feature, so no harm done, I guess.

Reply:
I left this out. If it ends up being needed, please lemme know. But it all works for me without it just fine.